docs/sphinx/index.rst

```diff
@@ -167,6 +171,9 @@ Part of the configuration can also be addressed using a json file. By default, i
   spack_activate       **None**                   Spack packages to activate                       **None**
  ==================== ========================== ================================================ =======================================
 
+However, Uberenv config can also be setup to sit in a location external to the uberenv directory itself. In particular when Uberenv is used as a submodule. Uberenv identifies such a case when no ``project.json`` file is found next to the script. It will then look for ``.uberenv_config.json`` recursively in parent directories. The typical usage is to place it at the root directory of the project.
```

Comment:
```suggestion
However, Uberenv configuration may instead sit in a location external to the uberenv directory itself. In particular when Uberenv is used as a submodule. Uberenv identifies such a case when no ``project.json`` file is found next to the script. It will then look for ``.uberenv_config.json`` recursively in parent directories. The typical usage is to place it at the root directory of the project.
```